arxiv/base/factory.py

```diff
@@ -13,4 +13,5 @@ def create_web_app() -> Flask:
     app.config.from_pyfile(os.path.join(basedir, 'config.py'))
     Base(app)    # Gives us access to the base UI templates and resources.
     app.register_blueprint(routes.blueprint)
+    print(app.url_map)
```

Comment:
Looks like this is left over from debuging :-)

Reply:
whoops! Fixed.